app/models/member.rb

```diff
@@ -235,32 +229,31 @@ def update_newsletter_subscription
     end
   end
 
-  def newsletter_subscribe(testing=false)
-    return true if (Rails.env.test? && !testing)
+  def newsletter_subscribe(testing = false)
+    return true if Rails.env.test? && !testing
     gb = Gibbon::API.new
-    res = gb.lists.subscribe({
-      id: Growstuff::Application.config.newsletter_list_id,
-      email: { email: email },
-      merge_vars: { login_name: login_name },
-      double_optin: false # they already confirmed their email with us
-    })
+    gb.lists.subscribe({
+                          id: Growstuff::Application.config.newsletter_list_id,
```

Comment:
Yep, I can turn off the parameter alignment check. That's around like 400 in the .rubocop.yml